ad-auth.txt

```diff
@@ -0,0 +1,3 @@
+Cøbra (https://bitcointalk.org/index.php?action=profile;u=249495) is authorized
+to sell advertising slots for this website (https://bitcoin.org).
+
```

Comment:
This should not be merged until such authorization has consensus. (Yes, this isn't a technical comment, but neither is this part of the change technical nor merely adding infrastructure.)